update-dependencies/CliDependencyHelper.cs

```diff
@@ -35,11 +49,12 @@ public static string GetSharedFrameworkVersion(string cliVersion)
             return sharedFrameworkVersion;
         }
 
-        private static string GetCommitHash(string cliVersion)
+        private string GetCommitHash()
         {
-            using (ZipArchive archive = DownloadCliInstaller(cliVersion).Result)
+            using (ZipArchive archive = DownloadCliInstaller().Result)
             {
-                ZipArchiveEntry versionTxtEntry = archive.GetEntry($"sdk\\{cliVersion}\\.version");
+                string zipEntryName = CliMajorVersion == 1 ? $"sdk/{CliVersion}/.version" : $"sdk\\{CliVersion}\\.version";
```

Comment:
Looking at the [ZipArchive.GetEntry implementation](https://github.com/dotnet/corefx/blob/2f554b13fe67c8cf9b37c70a711de1ba5575638c/src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs#L261), a simple Dictionary.TryGetValue is used and no transformation is performed on the FullName.  This leads me to conclude the path syntax must match the way the zip was constructed.